pkgs/development/arduino/arduino-mk/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "1.6.0";
+  name = "arduino-mk-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "sudar";
+    repo   = "Arduino-Makefile";
+    rev    = "${version}";
+    sha256 = "0flpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
```

Comment:
This hash won't be valid for fetchFromGitHub, change a character in it, like the l2x to l3x, and try building to get the correct hash.